presentation/src/main/java/me/androidbox/presentation/event/viewmodel/EventViewModel.kt

```diff
@@ -39,6 +39,10 @@ class EventViewModel @Inject constructor(
     private val _eventScreenState: MutableStateFlow<EventScreenState> = MutableStateFlow(EventScreenState())
     val eventScreenState = _eventScreenState.asStateFlow()
 
+    init {
+        Log.d("EVENT_VIEWMODEL", "THIS IS IN THE EVENT_VIEW_MODEL")
+        eventScreenState.value.eventId
```

Comment:
Nono
You should receive the savedStateHandle in the viewmodel constructor
And then, get the id as I showed you on my code